sport/app/football/views/matchList/matchesComponent.scala.html

```diff
@@ -1,25 +1,31 @@
+@import org.joda.time.LocalDate
 @import football.model.MatchesList
+@import model.Competition
+@import pa.FootballMatch
 @import views.support.`package`.Seq2zipWithRowInfo
+@import views.support.RowInfo
 
 @(matchesList: MatchesList, customLink: Option[(String, String)] = None)(implicit request: RequestHeader, context: model.ApplicationContext)
 
-<div data-component="football-matches-embed" class="c-football-matches football-embed">
+@template1(date: LocalDate, competitionMatches: List[(Competition, List[FootballMatch])], dateRow: RowInfo) = {
```

Comment:
New name for this template would be great too

Reply:
Done